kivy/uix/modalview.py

```diff
@@ -102,25 +112,16 @@ class ModalView(AnchorLayout):
     .. versionchanged:: 2.0.0
         Added property 'overlay_color'.
 
-    '''
+    """
 
-    auto_dismiss = BooleanProperty(True)
+    auto_dismiss = BooleanProperty()
```

Comment:
While it does default to True, it should be explicitly provided. It just makes it harder to follow if the default isn't provided and doesn't really help. So please add it back.

Reply:
Ok, although I personally prefer short/clean code, especially in cases like this, where the default value is **explicitly** documented.